### ISSUE TITLE
Still working?

### ISSUE THREAD

**Issue by lurace063** [alt-space-c/valorant-triggerbot#24]
Hey :)
Does it still work? I know it's new, but I just wanted to ask. Have a nice day :)

**Comment by ghost** [comment]
Hi
 
I don’t recommend downloading the entire file because two friends did and permanently banned them and hardware. What I did was copy the code of the two important files and so I did not receive any penalty, but the system did not work as autoclic because it does not read very well. In practical mode it reacted between 5mss and 20mss but in the beginning it reacted much later worse than the average human reaction.

In conclusion, create your own aimbot since triggerbot is well known and Riot already has defenses against this aimbot type.

best regards.
